src/mlpack/tests/cf_test.cpp

```diff
@@ -496,212 +496,37 @@ void Serialization()
 
 /**
  * Make sure that correct number of recommendations are generated when query
- * set for randomized SVD.
+ * set for all methods.
  */
-TEST_CASE("CFGetRecommendationsAllUsersRandSVDTest", "[CFTest]")
+TEMPLATE_TEST_CASE("CFGetRecommendationsAllUsersTest", "[CFTest]",
+  RandomizedSVDPolicy, RegSVDPolicy, BatchSVDPolicy, NMFPolicy,
+  SVDCompletePolicy, SVDIncompletePolicy, BiasSVDPolicy, SVDPlusPlusPolicy,
+  QUIC_SVDPolicy)
```

Comment:
```suggestion
    RandomizedSVDPolicy, RegSVDPolicy, BatchSVDPolicy, NMFPolicy,
    SVDCompletePolicy, SVDIncompletePolicy, BiasSVDPolicy, SVDPlusPlusPolicy,
    QUIC_SVDPolicy)
```
Simple style fix. :+1: